x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/endpoint_package_custom_extension.tsx

```diff
@@ -106,7 +106,13 @@ export const EndpointPackageCustomExtension = memo<PackageCustomExtensionCompone
       ]
     );
 
-    return canAccessEndpointManagement ? artifactCards : <NoPermissions />;
+    return loading ? (
+      <EuiLoadingSpinner data-test-subj="endpointExtensionLoadingSpinner" />
```

Comment:
Could you try using our [Management loader](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/management/components/management_page_loader.tsx) instead? Not sure how it will looks under the advanced tab but it should put the loader in the middle of the empty page.

Reply:
@dasansol92 We have [some code in that component ](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/management/components/management_empty_state_wrapper.tsx#L13)to center the loader on the page because its mean to really be used inside of an entire page.

Reply:
That's true, good point!